sentry-tracing/src/converters.rs

```diff
@@ -103,6 +103,36 @@ pub fn breadcrumb_from_event(event: &tracing_core::Event) -> Breadcrumb {
     }
 }
 
+fn contexts_from_event(
+    event: &tracing_core::Event,
+    event_tags: BTreeMap<String, Value>,
+) -> BTreeMap<String, sentry_core::protocol::Context> {
+    let event_meta = event.metadata();
+    let mut location_map = BTreeMap::new();
+    if let Some(module_path) = event_meta.module_path() {
+        location_map.insert("module_path".to_string(), module_path.into());
+    }
+    if let Some(file) = event_meta.file() {
+        location_map.insert("file".to_string(), file.into());
+    }
+    if let Some(line) = event_meta.line() {
+        location_map.insert("line".to_string(), line.into());
+    }
+
+    let mut context = BTreeMap::new();
+    context.insert(
+        "Rust Tracing Tags".to_string(),
+        sentry_core::protocol::Context::Other(event_tags),
+    );
+    if !location_map.is_empty() {
+        context.insert(
+            "Rust Tracing Location".to_string(),
```

Comment:
its quite unfortunate we don’t have any other event attrs for source location :-(